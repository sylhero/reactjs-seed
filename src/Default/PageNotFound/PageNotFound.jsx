import React from 'react';

const PageNotFound = () => {
    return <h1>Page not found</h1>;
};

export default PageNotFound;
