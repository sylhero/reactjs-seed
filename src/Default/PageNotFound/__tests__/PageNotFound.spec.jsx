import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../PageNotFound';

describe('PageNotFound', () => {
    it('should create a basic snapshot', () => {
        const wrapper = shallow(<PageNotFound />);
        expect(wrapper).toMatchSnapshot();
    });
});
