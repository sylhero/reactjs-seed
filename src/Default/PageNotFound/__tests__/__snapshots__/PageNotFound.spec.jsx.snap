// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should create a basic snapshot 1`] = `
<h1>
  Page not found
</h1>
`;
